### ISSUE TITLE
Feature Request: Download location 

### ISSUE THREAD

**Issue by 3nad90** [meganz/iOS#45]
## General Information

* MEGA Version:
4.9.3
* iOS Version(s):
 13.3
* Devices/Simulators:
Ipad 11 2018
* Reproducible? (Yes/No):
No

##  Can you please add choice to where I can download files on my ipad or my external drive?
thanks

**Comment by wizard86pz** [comment]
I do not think this is possible for any app on iOS. It's about iOS limitation, not on app side.
